builtin/credential/approle/path_role.go

```diff
@@ -780,7 +792,7 @@ func (b *backend) pathRoleCreateUpdate(req *logical.Request, data *framework.Fie
 		role.Period = time.Second * time.Duration(data.Get("period").(int))
 	}
 	if role.Period > b.System().MaxLeaseTTL() {
-		return logical.ErrorResponse(fmt.Sprintf("'period' of '%s' is greater than the backend's maximum lease TTL of '%s'", role.Period.String(), b.System().MaxLeaseTTL().String())), nil
+		return logical.ErrorResponse(fmt.Sprintf("period of %q is greater than the backend's maximum lease TTL of %q", role.Period.String(), b.System().MaxLeaseTTL().String())), nil
```

Comment:
This changes the quotes from single quotes to double quotes, if it matters.

Reply:
@calvn `%q` add the double quotes automatically.

Reply:
Right, however the old syntax printed the values wrapped around single quotes so just pointing that difference out :)

Reply:
Oh, I had misread your comment. Change from single to double quotes shouldn't matter IMO.

Reply:
The only potential change I see is that whenever this error gets returned back as JSON, the message will contain escaped quotes which may not look as clean (`"\"error value\""` vs `"'error value'"`).

Reply:
You are right. Its also true for a lot of places in Vault. It may not look as clean to humans, but parsing it and handling it is perfectly legal. I am inclined towards thinking that this is okay.